cloud/gcp/modules/database/main.tf

```diff
@@ -38,10 +39,10 @@ resource "google_sql_database_instance" "civiform_db" {
   }
 }
 
-## TODO(ktoor@google.com): Change this with the service account running the application.
+// Service accounts should be created with ".gserviceaccount.com" suffix.
```

Comment:
suggestion (blocking): move this comment to directly prior to L44 since that appears to be the piece of code it is addressing, rather than the entire resource block